worlds/ror2/__init__.py

```diff
@@ -102,9 +103,25 @@ def fill_slot_data(self):
 
     def create_item(self, name: str) -> Item:
         item_id = item_table[name]
-        item = RiskOfRainItem(name, True, item_id, self.player)
+        if name == "Dio's Best Friend":
+            prog = True
+        else:
+            prog = False
+        item = RiskOfRainItem(name, prog, item_id, self.player)
```

Comment:
```suggestion
        item = RiskOfRainItem(name, name == "Dio's Best Friend", item_id, self.player)
```

Slightly more concise way of doing this, if you like.